packages/webpack/index.js

```diff
@@ -31,11 +31,11 @@ function getFiles (stats, check) {
 
   return Object.keys(entries)
     .reduce((assets, i) => assets.concat(entries[i].assets), [])
-    .map(i => {
+    .map(({ name }) => {
       if (check.webpackConfig.output && check.webpackConfig.output.path) {
-        return join(check.webpackConfig.output.path, i)
+        return join(check.webpackConfig.output.path, name)
       } else {
-        return join(process.cwd(), 'dist', i)
+        return join(process.cwd(), 'dist', name)
```

Comment:
This was causing errors that the paths should be strings, but were objects. How did this ever work? I assume `name` was the original intention?

Also, I tried to refactor this into a cute ternary arrow function return, but it caused multiple ESLint errors as the rules that conflict with Prettier have not been disabled.